app/index.html

```diff
@@ -92,38 +92,63 @@ <h2 class="card-pf-title">
                 <div id="componentanalyses">
                   <form class="form-inline">
                     <div class="form-group">
-                      <label for="ecosystem">Ecosystem:</label>
-                      <input type="text" class="form-control" id="ecosystem">
+                      <input type="text" class="form-control" id="ecosystem" placeholder="Ecosystem">
                     </div>
                     <div class="form-group">
-                      <label for="component">Component:</label>
-                      <input type="text" class="form-control" id="component">
+                      <input type="text" class="form-control" id="component" placeholder="Component">
                     </div>
                     <div class="form-group">
-                      <label for="version">Version:</label>
-                      <input type="text" class="form-control" id="version">
+                      <input type="text" class="form-control" id="version" placeholder="Version">
                     </div>
-                    <button type="button" id="stackbtn" class="btn btn-default pull-right">Submit</button>
+                    <button type="button" id="stackbtn" class="btn btn-primary pull-right">Submit</button>
```

Comment:
The submit button is not working properly - I can't press enter in the form. Take a look at the waitlist form, I wired it up properly to respond to submits.